draft-ietf-scitt-architecture.md

```diff
@@ -877,11 +877,17 @@ Such policies may use as input all information in the Envelope, the Receipt, and
 
 # Privacy Considerations
 
-Transparency Services MAY support anonymous access.
-Issuers SHOULD ensure Signed Statements submitted to public access services are acceptable for public disclosure.
-Publicly accessible Signed Statements MUST NOT carry confidential information.
-Once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
-In some deployments, a special role, such as an Auditor, might require access to Signed Statements.
+Interactions with Transparency Services are expected to use appropriately strong encryption and authorization technologies.
+
+The Transparency Service is trusted with the confidentiality of the Signed Statements presented for Registration.
+Issuers are responsible for verifying that the Transparency Service's privacy and security posture is suitable for the contents of their Signed Statements prior to Registration.
```

Comment:
```suggestion
Clients are responsible for verifying that the Transparency Service's privacy and security posture is suitable for the contents of the Signed Statements they submit prior to Registration.
```

We draw a distinction between Clients and Issuers elsewhere, and even clarify "The Client (which is not necessarily the Issuer)", and the submission performed by the Client.

I don't think we say anything about the Client always acting faithfully/honestly on behalf of the Issuer. If that's always the case, then the current wording is fine. If not, perhaps this paragraph and the next ought to be reworded slightly, and a comment added to clarify that the Client is trusted in this process?

Or maybe I am overthinking this, and this is in fact clear to everyone :)

Reply:
There is a nuance between them, but I agree that Clients interact with the Transparency Service, and Issuers sign statements. Or, rather an Issuers signing material is used to sign statements, which "Clients" may operate upon.

Reply:
This introduces a 'turtles' situation where "Issuers are responsible for verifying that Clients that submit Signed Statements on their behalf are capable to evaluate the Transparency Service's privacy and security posture and only submit where such posture is suitable for the contents of the Signed Statements."

Reply:
@JAG-UK quite right, I have amended the proposed wording very slightly to say "Turtles", rather than "the top-most turtle". I am also happy to see this closed without further action.